ui/app/templates/storage/index.hbs

```diff
@@ -0,0 +1,233 @@
+              </B.Td>
+              <B.Td data-test-csi-volume-allocations>
+                {{B.data.allocationCount}}
+              </B.Td>
+            </B.Tr>
+          </:body>
+        </Hds::Table>
+        {{#if (gt this.sortedCSIVolumes.length this.userSettings.pageSize)}}
+          <Hds::Pagination::Numbered
+            @totalItems={{this.filteredCSIVolumes.length}}
+            @currentPage={{this.csiPage}}
+            @pageSizes={{this.pageSizes}}
+            @currentPageSize={{this.userSettings.pageSize}}
+            @onPageChange={{action this.handlePageChange "csi"}}
+            @onPageSizeChange={{action (mut this.userSettings.pageSize)}}
+          />
+        {{/if}}
+      {{else}}
+        <div class="empty-message" data-test-empty-csi-volumes-list-headline>
+          {{#if this.csiFilter}}
+            <p>No CSI volumes match your search for "{{this.csiFilter}}"</p>
+            <Hds::Button @text="Clear search" @color="secondary" {{on "click" (queue (action (mut this.csiFilter) "") (action this.handlePageChange "csi" 1))}} />
+          {{else}}
+            <p>No CSI Volumes found</p>
+          {{/if}}
+        </div>
+      {{/if}}
+    </Hds::Card::Container>
+
+    <Hds::Card::Container @level="base" @hasBorder={{false}} class="storage-index-table-card">
+      <header aria-label="Dynamic Host Volumes">
+        <h3>Dynamic Host Volumes</h3>
+        <p class="intro">
+          Storage provisioned via plugin scripts on a particular client, modifiable without requiring client restart.
+          <Hds::Link::Inline @href="https://developer.hashicorp.com/nomad/docs/commands/volume/create" @icon="docs-link" @iconPosition="trailing">Read more</Hds::Link::Inline>
+        </p>
+        <div class="search">
+          <Hds::Form::TextInput::Field
+            data-test-dynamic-host-volumes-search
+            @type="search"
+            @value={{this.dhvFilter}}
+            placeholder="Search Dynamic Host Volumes"
+            {{on "input" (action this.applyFilter "dhv")}}
+          />
+        </div>
+      </header>
+      {{#if this.sortedDynamicHostVolumes.length}}
+        <Hds::Table @caption="Dynamic Host Volumes"
+          @model={{this.sortedDynamicHostVolumes}}
+          @columns={{this.dhvColumns}}
+          @sortBy={{this.dhvSortProperty}}
+          @sortOrder={{if this.dhvSortDescending "desc" "asc"}}
+          @onSort={{action this.handleSort "dhv"}}
+        >
+          <:body as |B|>
+            <B.Tr data-test-dhv-row>
+              <B.Td>
+                <LinkTo data-test-dhv-name={{B.data.name}} @route="storage.volumes.dynamic-host-volume"
+                  @model={{B.data.idWithNamespace}}>
+                    {{B.data.name}}
+                  </LinkTo>
+              </B.Td>
+              {{#if this.system.shouldShowNamespaces}}
+                <B.Td>{{B.data.namespace}}</B.Td>
+              {{/if}}
+              <B.Td>{{B.data.node.name}}</B.Td>
```

Comment:
Is it feasible to link to the node from here?

Reply:
great suggestion